pytest.ini

```diff
@@ -0,0 +1,18 @@
+[pytest]
+testpaths = 
+    tests/client_test.py
+    tests/compare_test.py
+    tests/dtypes_tests.py
+    tests/groupby_test.py
+    tests/join_test.py
+    tests/operator_tests.py
+    tests/setops_test.py
+    tests/string_test.py
+    tests/where_test.py
```

Comment:
Just curious -- is the expectation that there will always be an explicit list, or do you expect this to eventually run everything in the `tests` dir?

Reply:
@ronawho good question and yeah, would recommend getting to a point that eventually we run everything in tests directory

Reply:
Here's the tests that I think are not currently being run:

 - check.py
 - coargsort.py
 - read_all_tests.py
 - read_write_tests.py


check.py might be a little bit different since it's used as the quick sanity check too. 

coargsort was recently added, but it'd be great to convert that too. (I'd be interested in doing that to familiarize myself with pytest a little more)

The other ones look like hdf5 tests that require a file, so are maybe also weird?

Reply:
@ronawho Cool, go for it on coargsort! I agree that should be supported. Also, the read tests will be updated in the near future with an hdf5 file generation utility.

Reply:
I included updating coargsort in https://github.com/mhmerrill/arkouda/compare/master...ronawho:pytest-test